### DISCUSSION TITLE
Trying to use MDX in Next.js runtime (@mdx-js/runtime not @mdnext/components)

### DISCUSSION THREAD

**Discussion by codercatdev** [vercel/next.js#19700] [Help]
Here is my original question
https://github.com/mdx-js/mdx/issues/1353

Which then lead to a working example in a react app.
https://codesandbox.io/s/great-mclean-j038y

However when I try to take this exact code over to Next.js it bombs and says 
```
./node_modules/@babel/core/lib/transformation/normalize-file.js
Module not found: Can't resolve 'fs' in '/sandbox/node_modules/@babel/core/lib/transformation'
```
I have tried looking at what Christian was referencing here, but it has no effect.
https://github.com/vercel/next.js/discussions/17897#discussion-24802

Any help would be greatly appreciated!

**Comment by kevinwgutierrez** [comment]
Hi there.

Did you ever find an answer for this?
